target/linux/ath79/dts/qca9531_mikrotik_routerboard-wapr-2nd.dts

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9533_mikrotik_routerboard-16m.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-wapr-2nd", "qca,qca9531";
+	model = "MikroTik RouterBOARD wAPR-2nD (wAP R)";
+
+	aliases {
+		led-boot = &led_wlan;
+		led-failsafe = &led_wlan;
+		led-upgrade = &led_wlan;
+		led-running = &led_wlan;
```

Comment:
rssihigh instead?

Reply:
These are the leds, wlan top right, rssihigh bottom right:

![image](https://user-images.githubusercontent.com/6009603/99387398-37d7cf00-28d4-11eb-85b3-9d4fbbf61e27.png)

I agree using rssihigh is better: it will be off when the wlan led is off, but not the other way round. I'm changing it.

Reply:
Of course, you might also use rssilow, as that's closer to the power symbol; eventually, it won't make much of a difference. Just Wifi is worse, as boot will conflict with the DT trigger for WiFi.
I assume the power led is not controllable via GPIO?

Reply:
Great, makes perfect sense!

Power LED doesn't seem to be controllable via GPIO, neither it is with RouterOS.